src/excel_adapter/excel_adapter.go

```diff
@@ -15,29 +15,20 @@ type ExcelAdapterConfig struct {
 	Parse    internals.ParseConfig
 }
 
-type ExcelAdapter struct {
-	config     ExcelAdapterConfig
-	boards     map[string]models.Board
-	globalInfo models.GlobalInfo
-	document   internalModels.Document
-}
-
-func NewExcelAdapter(config ExcelAdapterConfig) ExcelAdapter {
-	//TODO: poner config del download nesteado en el TOML
+func FetchBoardsAndGlobalInfo(config ExcelAdapterConfig) (map[string]models.Board, models.GlobalInfo) {
 	document := fetchDocument(config.Download, config.Parse)
 
-	return ExcelAdapter{
-		config: config,
-		boards: getBoards(document, config.Parse.AddressTable),
-		globalInfo: getGlobalInfo(document, GlobalInfoConfig{
-			AddressTable:    config.Parse.AddressTable,
-			BackendEntryKey: config.Parse.BackendEntryKey,
-			UnitsTable:      config.Parse.UnitsTable,
-			PortsTable:      config.Parse.PortsTable,
-			IdsTable:        config.Parse.IdsTable,
-		}),
-		document: document,
-	}
+	globalInfo := getGlobalInfo(document, GlobalInfoConfig{
```

Comment:
this is just me being picky, let me know your opinion, what about changing the parser config struct to be the three other fields and then this struct and in the config you have:

```toml
[excel.parse]
global_sheet_prefix = "GLOBAL "
board_sheet_prefix = "BOARD "
table_prefix = "[TABLE] "
global.address_table = "addresses"
global.backend_entry_key = "Backend"
global.units_table = "units"
global.ports_table = "ports"
global.ids_table = "ids"
```

(or changing global with any other identifier)

Reply:
Sure thing, i'll introduce this change in the next PR